src/Item_Devices.php

```diff
@@ -709,16 +713,18 @@ public static function showForItem(CommonGLPI $item, $withtemplate = 0)
                         $item->getName()
                     )
                 );
-                $link->getTableGroup(
-                    $item,
-                    $table,
-                    $table_options,
-                    $delete_all_column,
-                    $common_column,
-                    $specific_column,
-                    $delete_column,
-                    $dynamic_column
-                );
+                if ($link instanceof Item_Devices) {
```

Comment:
Maybe not needed anymore with the previous recommendation of not using `getType`.